schedule/yast/sle/flows/default_s390x_kvm.yaml

```diff
@@ -0,0 +1,61 @@
+---
+# Default ordered sequence of steps to be optionally overwritten for s390x
+bootloader:
+  - installation/bootloader_start
+setup_libyui:
+  - installation/setup_libyui
+access_beta:
+  - installation/access_beta_distribution
+product_selection:
+  - installation/product_selection/install_SLES
+license_agreement:
+  - installation/licensing/accept_license
+registration:
+  - installation/registration/register_via_scc
+extension_module_selection:
+  - installation/module_selection/select_nonconflicting_modules
```

Comment:
please, set default as in `default.yaml`

Reply:
changed.